libs/client/src/lib/client-example.ts

```diff
@@ -6,7 +6,7 @@
  *  Start API:
  *    nx serve api
  *  Command:
- *    DEBUG=tskmgr:* TS_NODE_PROJECT=libs/client/tsconfig.lib.json node --require ts-node/register --require tsconfig-paths/register libs/client/src/lib/client-example.ts
+ *    DEBUG=tskmgr:* ts-node --project libs/client/tsconfig.lib.json -r tsconfig-paths/register "libs/client/src/lib/client-example.ts"
```

Comment:
I prefer to stick with node command because ts-node is not supported when you want to attach the debugger as far as I can remember.

Reply:
the old command was not working in the upgraded environment and with the latest LTS node. Also, it is already used ts-node by calling this 'node --require ts-node/register', so it should not impact debug feature.